element-closest.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+/* globals window, Element */
+
+if (window.Element && !Element.prototype.closest) {
```

Comment:
This will break in node; please ensure it's wrapped in `typeof window !== 'undefined`, or, use `typeof Element !== 'undefined'` and reference it as a global.